packages/resource-deployment/scripts/add-tags-for-batch-vmss.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set -eo pipefail
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -v <vmss name> -r <vmss resource group> -b <batch resource group name>
+"
+    exit 1
+}
+
+addResourceGroupNameTagToVMSS(){
+    az resource update --set tags.ResourceGroupName="$resourceGroupName" -g "$vmssResourceGroup" -n "$vmssName" --resource-type "Microsoft.Compute/virtualMachineScaleSets"
```

Comment:
can you call the new script you created for adding a tag instead of this line?

Reply:
this is the only file script i have for tags.

Reply:
yes that's the one I mean, that script for tags runs exactly the same command here